src/App.js

```diff
@@ -2,59 +2,93 @@
 import React, { useEffect, useState } from 'react';
 import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
+import TodoListItem from './TodoListItem';
 
 
 function App() {
 
   const [todoList, setTodoList] = useState ([])
   const [isLoading, setIsLoading] = useState (true)
+  const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`
 
-  useEffect(() => {
-    new Promise((resolve, reject) => {
-      setTimeout(() => {
-        resolve(
-          {data: {
-          todoList: JSON.parse(localStorage.getItem("savedTodoList")) || []
-        }}
-        )
-      }, 2000)
-    })
-    .then((result) => {
-      setTodoList(result.data.todoList)
-      setIsLoading(false)
+  const fetchData = async () => {
+    const options = {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+      }
     }
-    )
-  }, [])
 
-  useEffect (() => {
-    if (!isLoading)  {
-      localStorage.setItem("savedTodoList", JSON.stringify(todoList))
+    try {
+      const response = await fetch (url, options)
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`)
+      }
+      const data = await response.json()
+      const todos = data.records.map((todo) => {
+        return {
+          title: todo.fields.title,
+          id: todo.id
+        }
+
+      })
+      setTodoList(todos)
+      setIsLoading(false)
+
+    } catch (error) {
+      console.log(error)
     }
-  }, [todoList])
+  }
+
+  useEffect(() => {
+    fetchData()
+  }, [])
 
   const addTodo = (newTodo) => {
-    setTodoList((prevList) => [...prevList, newTodo]);
-  };
+    fetch (url, {
+      method: 'POST',
+      headers: {  
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+    },
+      body: JSON.stringify({fields: newTodo})
+    }
+    )
+    .then (response => response.json())
+    .then (data => { 
+      const addTodo = 
+       {
+        title: data.fields.title,
+        id: data.id
+      }
+      setTodoList((prevList) => [...prevList, addTodo])
+    })
+    }
 
 
-  function removeTodo (todo) {
-    setTodoList((newArray) => newArray.filter((list) => todo.id !== list.id))
+  const removeTodo = async (todo) => {
+    console.log(todo)
+    const urlDel = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/` + todo.id
```

Comment:
You could actually include the `todo.id` in the template string like this:
```suggestion
    const urlDel = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/${todo.id}`
```